packages/@ember/-internals/glimmer/index.ts

```diff
@@ -400,7 +400,6 @@ export { default as AbstractComponentManager } from './lib/component-managers/ab
 // TODO just test these through public API
 // a lot of these are testing how a problem was solved
 // rather than the problem was solved
-export { INVOKE } from './lib/helpers/mut';
```

Comment:
This is used as intimate API by a few addons, most notably `ember-concurrency`.

https://github.com/machty/ember-concurrency/blob/067fb79524b69ba4118765bb709ef93aec4800bc/addon/utils.js#L75-L90

We need a plan for how to deal with those downstream issues.